packages/react-components/src/components/ParticipantsButton.tsx

```diff
@@ -121,6 +121,11 @@ export interface ParticipantsButtonProps extends ControlBarButtonProps {
    * Callback to render the context menu for each participant
    */
   onRemoveParticipant?: (userId: string) => void;
+  /* @conditional-compile-remove(raise-hands) */
+  /**
+   * Callback to render the context menu for each participant
+   */
+  onLowerHands?: (userId: string[]) => void;
```

Comment:
I feel like this is just for lowering one participant hand, can we name it as onLowerHand? And make a handler to match this funciton name